conf/standalone/log4j.xml

```diff
@@ -3,7 +3,7 @@
 <log4j:configuration>
 
   <appender name="FileRoll" class="org.apache.log4j.rolling.RollingFileAppender">
-    <param name="file" value="${gobblin.logs.dir}/standalone.out" />
+    <param name="file" value="${gobblin.logs.dir}/gobblin-current.log" />
```

Comment:
file name here is mode name, any reason to change it to gobblin-current ?

Reply:
Ah good point I'll just change it back, I think I wanted it to match prior to the script change but I realize now there's no real point of breaking naming convention